tests/org.eclipse.xtext.xbase.testlanguages/src/org/eclipse/xtext/xbase/testlanguages/GenerateXbaseTestLanguages.mwe2

```diff
@@ -1,11 +1,10 @@
-module org.eclipse.xtext.xbase.testlanguages.GenerateXImportSectionTestLang
+module org.eclipse.xtext.xbase.testlanguages.GenerateXbaseTestLanguages
 
 import org.eclipse.emf.mwe.utils.*
 import org.eclipse.xtext.generator.*
 import org.eclipse.xtext.ui.generator.*
 
-var grammarURI = "classpath:/org/eclipse/xtext/xbase/testlanguages/XImportSectionTestLang.xtext"
-var fileExtensions = "__ximportimporttestlang"
+var fileExtensions = ""
```

Comment:
Ok - it seems to be too inconvenient to change the file extension per language without auto-inject. So the best choice seems to be the removed __ prefix.